CoolCurrencyWatchApp WatchKit Extension/InterfaceController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  InterfaceController.swift
+//  CoolCurrencyWatchApp WatchKit Extension
+//
+//  Created by Vincent Moyo on 2021/11/29.
+//
+
+import WatchKit
+import Foundation
+import UIKit
+
+class InterfaceController: WKInterfaceController {
+    
+    override func awake(withContext context: Any?) { }
+    override func willActivate() { }
+    override func didDeactivate() { }
+    
+    func alertAction(for currency: String) -> WKAlertAction {
+        let rateController = { self.pushController(withName: "rates", context: currency) }
+        return WKAlertAction(title: currency, style: .default, handler: rateController)
+    }
+    
+    @IBAction func exchangeRatePressed() {
+        
+        let poundCurrencyAction = alertAction(for: "Pound")
```

Comment:
Rather create these dynamically, no need to declare them like this. Happy if you sort of have an array with all the currencies and then create the alert actions from the array.